spec/cucumber/core/test/filters/locations_filter_spec.rb

```diff
@@ -120,20 +120,93 @@ def test_case_named(name)
           expect(receiver.test_case_locations).to eq [test_case_named('two').location]
         end
 
-        it "doesn't match a location after the scenario line" do
+        it "matches the location of a step of the scenario" do
           location = Test::Location.new(file, 9)
           filter = Test::LocationsFilter.new([location])
           compile [doc], receiver, [filter]
-          expect(receiver.test_case_locations).to eq []
+          expect(receiver.test_case_locations).to eq [test_case_named('two').location]
+        end
+
+        it 'matches a location on the last step of the scenario' do
+          location = Test::Location.new(file, 10)
+          filter = Test::LocationsFilter.new([location])
+          compile [doc], receiver, [filter]
+          expect(receiver.test_case_locations).to eq [test_case_named('two').location]
+        end
+
+        xit "matches a location on the scenario's comment" do
```

Comment:
I don't like this personally because this is technically ambiguous. Comments in the gherkin / pickle e.t.c. are attributed to a line, but not directly attached to a scenario. Whereas a tag **is** directly attached to a gherkin keyword.

Reply:
I'm with you on this one. I was honestly surprised to see this behavior was ever a part of cucumber. Since it doesn't pass anyways, I'll rm it from this PR, along with the other pending scenarios.